iothub/device/src/AmqpTokenRefresher.cs

```diff
@@ -10,68 +10,62 @@ namespace Microsoft.Azure.Devices.Client
     using Microsoft.Azure.Amqp;
     using System.Diagnostics;
     using Microsoft.Azure.Devices.Shared;
+    using Microsoft.Azure.Devices.Client.Transport;
 
-    internal sealed class IotHubTokenRefresher
+    internal sealed class AmqpTokenRefresher : IDisposable
     {
         private static readonly string[] AccessRightsStringArray = 
             AccessRightsHelper.AccessRightsToStringArray(AccessRights.DeviceConnect);
 
         static readonly TimeSpan BufferPeriod = TimeSpan.FromSeconds(120);
 
-        private readonly AmqpSession amqpSession;
+        private readonly AmqpClientSession amqpClientSession;
         private readonly IotHubConnectionString connectionString;
         private readonly string audience;
         private readonly CancellationTokenSource cancellationTokenSource;
         private volatile bool taskCancelled;
 
-        public IotHubTokenRefresher(AmqpSession amqpSession, IotHubConnectionString connectionString, string audience)
+        public AmqpTokenRefresher(AmqpClientSession amqpClientSession, IotHubConnectionString connectionString, string audience)
         {
-            this.amqpSession = amqpSession ?? throw new ArgumentNullException("amqpSession");
+            this.amqpClientSession = amqpClientSession ?? throw new ArgumentNullException("amqpClientSession");
             this.connectionString = connectionString;
             this.audience = audience;
             this.cancellationTokenSource = new CancellationTokenSource();
         }
 
-        public void Cancel()
+        internal void Cancel()
         {
```

Comment:
Adding a comment to provide deprecated API changes for any class that changed public to smaller scope.